deploying.md

```diff
@@ -0,0 +1,113 @@
+# Deploying Philanthropy Data Commons software
+
+This document guides one through deployment for the back-end services at
+https://api.philanthropydatacommons.org and
+https://auth.philanthropydatacommons.org.
+
+The front-end at https://pilot.philanthropydatacommons.org has its own process.
```

Comment:
Where is that process documented?  We might as well point to it from here.

Reply:
:thinking: :construction:

Reply:
I put a best guess link in for now, but I await a pointer from @reefdog or @jasonaowen regarding the deployment docs there.